File: parrot.java
import java.util.*;

public class Parrot {
    // Attributes
    List<String> words_ = new ArrayList<>();
    Random rand = new Random();
    int number_;

    // Constructor
    public Parrot(String word, int number) {
        words_.add(word);
        number_ = number;
    }

    public void say(){
        int k = rand.nextInt(words_.size());
        for(int i = 0; i < number_; i++)
            System.out.println(words_.get(k));
    }

    public void changeWord(String newWord, int number){
        words_.add(newWord);
        number_ = number;
    }


    public static void main(String[] args) {
        Parrot parrot = new Parrot("aaaa", 2);
        parrot.say();
        parrot.changeWord("dsds", 3);
        parrot.say();
    }
}

File: parrot.cpp
#include<iostream>
#include<vector>
#include<cstdlib>
#include<ctime>

using namespace std;

class Parrot{
    int number;
    vector<string> words;

    public:
        Parrot(string word, int num){
            this->words.push_back(word);
            number = num;
        }

    void changeWord(string word, int num){
        number = num;
        words.push_back(word);
    }

    void say(){
        srand(time(0));
        int k = rand()%words.size();
        for(int i = 0 ;i < number; i++){
            cout << words[k] << endl;
        }
    }
};

int main(){
    Parrot p("aaa", 2);
    p.say();
    p.changeWord("zzz", 3);
    p.say();
    return 0;
}